ushahidi/platform-client-mzima
# fix empty space for private categories

Fixes [#4743](https://github.com/ushahidi/platform/issues/4743)

# Summary
I added a conditional for the `category` div to render a tag from the returned array of categories only when `category.tag` is not empty.

## How to test the PR
1. Login to your deployment.
2. Click on "Settings".
3. Click on "Categories".
4. Create a category, set the visibility to "Only me".
5. Go back to "Settings".
6. Select "Surveys".
7. Create a survey or edit an old one.
8. Click on "Add new field".
9. Select "Categories".
10. Select the category you just created.
11. Save the survey.
12. Add a new post to the survey.
13. Make sure the category you added is selected.
14. Click submit post.
15. Go to the 'feed' view.
16. Publish the post.
17. Log out.
18. Go to the 'feed' view.
19. Locate the post you just created and click on it.
20. Compare the behavior you see to the one in the screenshot in issue [#4743](https://github.com/ushahidi/platform/issues/4743)
